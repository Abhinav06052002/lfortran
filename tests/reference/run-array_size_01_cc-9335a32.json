{
    "basename": "run-array_size_01_cc-9335a32",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_size_01_cc.f90",
    "infile_hash": "e61579eb96943f206c9157ca626df80ce4ea7c32281c0e979b27d943",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_size_01_cc-9335a32.stdout",
    "stdout_hash": "a779ee4f7341ddce58a41cee3d249ecf1b37216d7d4f16eb4e0f528e",
    "stderr": "run-array_size_01_cc-9335a32.stderr",
    "stderr_hash": "36f12657bb9f52a26ae1b18bca0578405c3f548f9793d07610781ae9",
    "returncode": 1
}