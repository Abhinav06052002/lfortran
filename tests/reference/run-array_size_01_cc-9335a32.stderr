semantic error: Incorrect number of arguments passed to the 'size' intrinsic. It accepts at least 1 and at most 3 arguments.
 --> tests/errors/array_size_01_cc.f90:5:14
  |
5 |     size_a = size(a, 1, 4, kind=4)
  |              ^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Incorrect number of arguments passed to the 'size' intrinsic. It accepts at least 1 and at most 3 arguments.
 --> tests/errors/array_size_01_cc.f90:6:14
  |
6 |     size_a = size()
  |              ^^^^^^ 
